### ISSUE TITLE
Add screenReaderFocusable attribute support

### ISSUE THREAD

**Issue by Irineu333** [Irineu333/SpeakTouch#120]
The attribute `screenReaderFocusable` signals, just like focusable, that the element should be focused by the screen reader, even if it does not have content, in this case reading its children [^1]

[^1]: https://developer.android.com/guide/topics/ui/accessibility/principles#content-groups

**Comment by Irineu333** [comment]
From what I understand, the `isScreenReaderFocusable` is similar to `isFocusable`, but exclusive to screen readers. Did you understand it in the same way?" @PatrykMis
